File: 0884-uncommon-words-from-two-sentences/0884-uncommon-words-from-two-sentences.go
import (
  "strings"
)
func uncommonFromSentences(s1 string, s2 string) []string {
  m := map[string]int{}
  arr := strings.Split(s1 + " " + s2, " ")
  for _, s := range arr {
    m[s]++;
  }
  ans := []string{}
  for word, count := range m {
    if count == 1 {
      ans = append(ans, word)
    }
  }
  return ans;
}

File: 0884-uncommon-words-from-two-sentences/0884-uncommon-words-from-two-sentences.ts
function uncommonFromSentences(s1: string, s2: string): string[] {
  const m: Record<string, number> = {};
  for (const word of (s1 + ' ' + s2).split(' ')) {
    if (m.hasOwnProperty(word)) {
      m[word]++;
    } else {
      m[word] = 1;
    }
  }
  const ans: string[] = [];
  for (const word in m) if (m[word] === 1) ans.push(word);
  return ans;
};